ocaml/runtime/caml/isa.h

```diff
@@ -66,7 +66,7 @@ CAML_REQUIRE_ARCH_PREFETCHW
 #endif
 
 #ifdef __PREFETCHWT1__
-CAML_REQUIRE_ARCH_PREFETCHWT1
+// CAML_REQUIRE_ARCH_PREFETCHWT1
```

Comment:
Is it part of this pull request on purpose?

Reply:
(looks like I was commenting on something not in the diff... sorry for the noise)